airsonic-main/src/main/java/org/airsonic/player/service/PlayerService.java

```diff
@@ -118,9 +125,9 @@ public synchronized Player getPlayer(HttpServletRequest request, HttpServletResp
         if (player == null) {
             player = new Player();
             createPlayer(player);
-//            LOG.debug("Created player " + player.getId() + " (remoteControlEnabled: " + remoteControlEnabled +
-//                      ", isStreamRequest: " + isStreamRequest + ", username: " + username +
-//                      ", ip: " + request.getRemoteAddr() + ").");
+//          LOG.debug("Created player " + player.getId() + " (remoteControlEnabled: " + remoteControlEnabled +
+//              ", isStreamRequest: " + isStreamRequest + ", username: " + username +
+//              ", ip: " + request.getRemoteAddr() + ").");
         }
```

Comment:
Please discard this change.

Reply:
Forgot to reset that after I'd used it for debug - will do.